graphql/__mocks__/ioredis.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-env jest */
+const ioredisMock = jest.requireActual('ioredis-mock')
```

Comment:
I'm a little confused by the purpose of this. The `requireActual` is getting the underlying module and then exporting it again, which should have no effect. Do you mean to create a mock from the module?

Reply:
From how I understand having this as `ioredis.js` means that the ioredis package is being mocked, and I'm basically replacing it's full implementation with `ioredis-mock` instead.

Reply:
Forgive my poor skimming here. I managed to miss the `-mock`! 🙄

Reply:
No worries!